packages/selinux-policy/selinux-policy/policy/constraints

```diff
@@ -25,6 +25,11 @@
 # name_list : name | name_list name		
 #
 
+# SRS: uncomment once you have SECMARK rules in-place
+# only allow kernel_t to send and receive unlabeled packets
+#constrain packet { send recv }
+#          ((t1 == kernel_t) or ((t2 != unlabeled_t) ));
```

Comment:
This is still commented out. Is that the intent?

Reply:
Missed this in my first set of replies.

I think it should remain commented out, or else networking won't work until a complete set of firewall rules is written which can be deep into the development process.

Alternatively, we could make some other exception based on an attribute and use a conditional to control access. That might be more generally desirable.